scraping-task-solution/SalahEl-Din_sln.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3 
+
+from bs4 import BeautifulSoup
+import requests
+URL = "https://commits.top/egypt.html"
+
+def get_data(LINK):
+    r = requests.get(LINK,"lxml")
+    soup = BeautifulSoup(r.content)
+    allSpans = soup.find_all("span")
+    allImgs = soup.find_all("img")
+    for span in allSpans:
+        if "p-name" in str(span):
+            pName = span
+        if "text-bold color-text-primary" in str(span):
+            noOfFollowers=span
+    for img in allImgs:
+        if "avatar" in str(img):
+            pImg = img["src"]
+    currentName = str(pName.contents).split("\\n")[1]
+    noOfFollowers = str(noOfFollowers.contents).strip("[']")
+    return currentName,noOfFollowers,pImg
+
+current_URL = URL
+r=requests.get(current_URL,"lxml")
+soup= BeautifulSoup(r.content) #content => byte string
+people_list = soup.find_all("td")
+print("Please wait while data is being scraped/stolen")
+for i in range(256):
```

Comment:
what is 265? it's called magic number. you 'd better define it as a constant 
NUM_OF_PEOPLE = 256

Reply:
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad